_pages/fr/roadmap.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+title:  "Roadmap"
+permalink: /lang/roadmap
```

Comment:
since this is the FR version, please already replace "lang" with "fr" (same for the lang: property below)